### PR TITLE
JP-33: feat: weather

### PR DESCRIPTION
**Pull request by Ixpiria**

### CODE REVIEW

File: frontend/pages/journey/[id].vue [hunk 1]
```diff
@@ -676,21 +753,135 @@ const handleUpload = (result: string) => {
...
-                                        />
                                         <div
-                                            class="absolute bottom-4 right-2 z-40 ml-10 flex h-16 w-16 items-center justify-center self-center rounded-full border-2 border-dashed border-natural-400 pl-1.5 pr-1.5 text-xs text-natural-400 dark:border-natural-50 dark:text-natural-50"
+                                            class="absolute bottom-4 right-2 z-40 ml-10 flex h-16 w-16 cursor-pointer select-none items-center justify-center self-center rounded-full border-2 border-dashed border-natural-400 pl-1.5 pr-1.5 text-xs text-natural-400 dark:border-natural-50 dark:text-natural-50"
                                         >
                                             <T key-name="journey.turn" />
                                         </div>
-                                        <SvgStripes
-                                            class="z-0 md:w-2/3 lg:w-1/2"
+                                        <SvgOtherStripes
```
Comment (stefll) [inline frontend/pages/journey/[id].vue]: Can u make this line shorter please 
image

File: frontend/components/SmallWeather.vue [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
+const props = defineProps({
+    maxTemp: { type: Number, required: true },
+    minTemp: { type: Number, required: true },
+    qrCode: { type: String, required: true },
```
Comment (EinSev) [inline frontend/components/SmallWeather.vue]: not needed

File: frontend/pages/journey/[id].vue [hunk 3]
```diff
@@ -676,21 +753,135 @@ const handleUpload = (result: string) => {
...
+                                            class="absolute bottom-1 z-0 -ml-1 w-32 self-start pb-2 md:w-2/3 lg:w-1/2"
                                         />
+                                        <div class="flex h-full">
+                                            <div class="z-0 ml-6 h-full w-1/2">
```
Comment (stefll) [inline frontend/pages/journey/[id].vue:763]: Remove the stripes (mobile version)
image

File: frontend/pages/journey/[id].vue [hunk 4]
```diff
@@ -676,21 +753,135 @@ const handleUpload = (result: string) => {
...
+                                                <div
+                                                    class="-ml-14 mt-3 flex h-1/2 items-center justify-center"
+                                                >
+                                                    <WeatherIcon
+                                                        class="-mt-2 ml-8 w-24"
```
Comment (stefll) [inline frontend/pages/journey/[id].vue:768]: please make this button a little bit smaller and move it to the right more (mobile version: front side and back side please)
image

File: frontend/pages/journey/[id].vue [hunk 5]
```diff
@@ -676,21 +753,135 @@ const handleUpload = (result: string) => {
...
+                                                        :weather-code="
+                                                            weatherCodeToday
+                                                        "
+                                                    />
```
Comment (stefll) [inline frontend/pages/journey/[id].vue:772]: The separators should be a bit longer and please align the the temperature 
image

Comment (stefll) [reply frontend/pages/journey/[id].vue:772]: You probably have to make the overcast icon slightly bigger. Bc we have the same problem here with another triü 
image

File: frontend/pages/journey/[id].vue [hunk 6]
```diff
@@ -676,21 +753,135 @@ const handleUpload = (result: string) => {
...
+                                                </div>
+                                                <div class="flex">
+                                                    <div>
+                                                        <div
+                                                            class="mt-1 flex h-full w-full items-center justify-start text-5xl text-text dark:text-natural-50"
```
Comment (stefll) [inline frontend/pages/journey/[id].vue]: image
can u move the description more to the middle so its under the temperature, please

File: frontend/components/WeatherIcon.vue
Comment (EinSev) [inline frontend/components/WeatherIcon.vue:1]: Dark mode icons are not used

File: frontend/pages/journey/[id].vue (hunk 5 above)
Comment (stefll) [reply frontend/pages/journey/[id].vue:772]: Or maybe another way (so u dont have to change the size of the icon for each icon)
image

File: frontend/pages/journey/[id].vue [hunk 7]
```diff
@@ -4,6 +4,7 @@ import { differenceInDays, differenceInHours, format } from "date-fns";
 import JSConfetti from "js-confetti";
 import QRCode from "qrcode";
 import resolveConfig from "tailwindcss/resolveConfig";
+import SmallWeather from "~/components/SmallWeather.vue";
```
Comment (EinSev) [inline frontend/pages/journey/[id].vue]: import needed? Nuxt should auto import

**Review by stefll — CHANGES_REQUESTED** [review]
Design: Looks good but still some minor things to change

File: frontend/pages/journey/[id].vue (hunk 3 above)
Comment (stefll) [reply frontend/pages/journey/[id].vue:763]: image
So it looks like this
Also check the color of the line. its not the right color

**Review by EinSev — CHANGES_REQUESTED** [review]

**Review by EinSev — DISMISSED** [review]

**Review by EinSev — APPROVED** [review]

**Review by stefll — APPROVED** [review]
Design Review: Looks good good

**Pull request merged**